telegram/dice.py

```diff
@@ -37,7 +37,7 @@ class Dice(TelegramObject):
         emoji (:obj:`str`): Emoji on which the dice throw animation is based.
 
     Args:
-        value (:obj:`int`): Value of the dice, 1-6.
+        value (:obj:`int`): Value of the dice, 1-6 for DICE and DARTS, 1-5 for BASKETBALL
```

Comment:
This should rather go in the `Note` above. It would also be good to describe the different values for the basketball. What I found out so far:

1 bounces off directly
2 circles & then bounces off
3 get's stuck between ring and the back thingy
4 circles, but get's in the net (stuck)
5 get's in the net directly
6 doesn't seem to exist.

Just use proper formulations instead :D
Please make sure that it's clear that Telegram didn't document that.

Reply:
Thanks for the review, I added the requested changes and fixed the docs.